### DISCUSSION TITLE
error during insalling

### DISCUSSION THREAD

**Discussion by Mirzaei97** [Xiangyu-Hu/SPHinXsys#336] [Q&A] [answered]
Hello
Good time
I encountered this error when I was installing the software SPHinXsys on Linux
Do you know a solution for it?

error: building lapack-reference:x64-linux failed with: BUILD_FAILED Please ensure you're using the latest port files with `git pull` and `vcpkg update`. Then check for known issues at:     https://github.com/microsoft/vcpkg

**Comment by Mirzaei97** [comment]
I tried this software with both Linux 20-04 and Linux 22-04 and both gave errors.

**Reply by Xiangyu-Hu** [reply, marked as answer]
It seems that you have not install gfortran in your system.
You can install it by apt install.

**Reply by Mirzaei97** [reply]
Thank you for your guidance

**Comment by Mirzaei97** [comment]
Hello
good time
I got this error while running the example in SPHinXsys
Do you know a solution for that?


Total Test time (real) = 1990.78 sec

The following tests did not run:
	 75 - bending_circular_plate.parametric_dp (Disabled)
	 95 - roof_under_self_weight.parametric_dp (Disabled)
	 96 - roof_under_self_weight.lattice (Disabled)

The following tests FAILED:
	 93 - test_3d_roof (Child aborted)
	 94 - roof_under_self_weight.dp_4 (Failed)
	 97 - test_3d_roof_parametric_cvt (Failed)
	 98 - test_3d_self_contact (Failed)
	 99 - test_3d_shell_particle_relaxation (Child aborted)
	100 - sphere_compression.half_sphere (Failed)
	101 - test_3d_shell_stability_half_sphere (Failed)
	102 - test_3d_slender_beam (Child aborted)
	103 - test_3d_stfb (Child aborted)
	104 - test_3d_stlw (Child aborted)
	105 - test_3d_taylor_bar (Failed)
	106 - test_3d_thin_plate (Child aborted)
	107 - test_3d_twisting_column (Child aborted)
